src/game/sound_init.c

```diff
@@ -293,7 +295,20 @@ void stop_shell_music(void) {
 /**
  * Called from threads: thread5_game_loop
  */
+
+#ifdef NO_CAP_MUSIC_RESTART
+static s8 sDoResetMusic = FALSE;
+extern void stop_cap_music(void);
+#endif
+
 void play_cap_music(u16 seqArgs) {
+#ifdef NO_CAP_MUSIC_RESTART
+    if (sDoResetMusic) {
+        sDoResetMusic = FALSE;
+        stop_cap_music();
+    }
+#endif
```

Comment:
this is confusing, if the define is NO_CAP_MUSIC_RESTART, whats this for? doing it manually?

Reply:
Sometimes cap music has to be restarted - when music is fading out if cap is close to expiring, cap music gets restarted otherwise it would just be silent if nothing is done. If you have a better idea how to name this variable, go for it :)

Reply:
Fixed conflicts.